### PR TITLE
auto convert JsonNode to Python dictionary, fixes #147

### LINKED ISSUE #147: Export Nim JsonNode as Python dict
When I pass Python `dict` to a Nim function I get a `JsonNode` (`dict -> JsonNode`). It would be great if it worked the other way around too (`JsonNode -> dict`).

### PR DESCRIPTION
**Pull request by Vindaar**
This fixes issue #147.

I'm not sure if this is what we want to do by default. @yglukhov you decide that. :)

### CODE REVIEW

**Comment by rominf** [comment]
Looks nice!

File: nimpy.nim [hunk 1]
```diff
@@ -801,6 +804,32 @@ proc nimObjToPy[T](o: T): PPyObject =
...
+    result = nimValueToPy(node.getFloat)
+  of JBool:
+    result = nimValueToPy(node.getBool)
+  of JString:
+    result = nimValueToPy(node.getStr)
+  of JArray:
+    result = pyLib.PyList_New(node.len)
+    for i in 0 ..< node.len:
+      let o = nimJsonToPy(node[i])
+      discard pyLib.PyList_SetItem(result, i, o)
```
Comment (yglukhov) [inline nimpy.nim:823]: Should be a `decref o` here?

Comment (Vindaar) [reply nimpy.nim:824]: Nope. I was confused when I saw your comment, but checked again. `PyList_SetItem` steals the reference. See the docs here:

https://docs.python.org/3.3/c-api/list.html?highlight=m#PyList_SetItem

**Comment by yglukhov** [comment]
Nice! Thank you!

**Pull request merged**